tests/test_dcrModel.py

```diff
@@ -165,64 +175,85 @@ def makeDummyWcs(self, rotAngle, pixelScale, crval, flipX=True):
         wcs = afwGeom.makeSkyWcs(crpix=crpix, crval=crval, cdMatrix=cdMatrix)
         return wcs
 
-    def makeDummyVisitInfo(self, azimuth, elevation):
+    def makeDummyVisitInfo(self, azimuth, elevation, exposureId=12345, randomizeTime=False):
         """Make a self-consistent visitInfo object for testing.
 
-        For simplicity, the simulated observation is assumed
-        to be taken on the local meridian.
-
         Parameters
         ----------
         azimuth : `lsst.geom.Angle`
             Azimuth angle of the simulated observation.
         elevation : `lsst.geom.Angle`
             Elevation angle of the simulated observation.
+        exposureId : `int`, optional
+            Unique integer identifier for this observation.
+        randomizeTime : `bool`, optional
+            Add a random offset to the observation time.
 
         Returns
         -------
         `lsst.afw.image.VisitInfo`
             VisitInfo for the exposure.
         """
-        lsstLat = -30.244639*degrees
-        lsstLon = -70.749417*degrees
-        lsstAlt = 2663.
-        lsstTemperature = 20.*u.Celsius  # in degrees Celcius
+        lsstLat = -30.244639*u.degree
+        lsstLon = -70.749417*u.degree
+        lsstAlt = 2663.*u.m
+        lsstTemperature = 20.*u.Celsius
         lsstHumidity = 40.  # in percent
         lsstPressure = 73892.*u.pascal
-        lsstWeather = Weather(lsstTemperature.value, lsstPressure.value, lsstHumidity)
-        lsstObservatory = Observatory(lsstLon, lsstLat, lsstAlt)
-        airmass = 1.0/np.sin(elevation.asRadians())
-        era = 0.*radians  # on the meridian
-        zenithAngle = 90.*degrees - elevation
-        ra = lsstLon + np.sin(azimuth.asRadians())*zenithAngle/np.cos(lsstLat.asRadians())
-        dec = lsstLat + np.cos(azimuth.asRadians())*zenithAngle
-        visitInfo = afwImage.VisitInfo(era=era,
-                                       boresightRaDec=afwGeom.SpherePoint(ra, dec),
-                                       boresightAzAlt=afwGeom.SpherePoint(azimuth, elevation),
-                                       boresightAirmass=airmass,
-                                       boresightRotAngle=0.*radians,
-                                       observatory=lsstObservatory,
-                                       weather=lsstWeather
-                                       )
+        loc = EarthLocation(lat=lsstLat,
+                            lon=lsstLon,
+                            height=lsstAlt)
+        airmass = 1.0/np.sin(elevation.asDegrees())
+
+        time = Time(2000., format='decimalyear', scale='tai', location=loc)
```

Comment:
Can you define this with `format="jyear"` so that you don't need to fiddle with the 0.5 day correction next line?
```python
>>> Time(2000.0, format="jyear", scale="tai").isot
'2000-01-01T12:00:00.000'
```